generate/elm.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Could we rename this directory to make it clear what is being generated? And maybe add a readme if we feel that it is warranted? It could have some instructions for how to generate a new practice exercise from the canonical data for example.